hbase-client/src/main/java/org/apache/hadoop/hbase/ClientMetaTableAccessor.java

```diff
@@ -257,7 +259,37 @@ private static CompletableFuture<Void> scanMeta(AsyncTable<AdvancedScanResultCon
     }
 
     CompletableFuture<Void> future = new CompletableFuture<Void>();
-    metaTable.scan(scan, new MetaTableScanResultConsumer(rowUpperLimit, visitor, future));
+    // Get the region locator's meta replica mode.
+    CatalogReplicaMode metaReplicaMode = CatalogReplicaMode.fromString(metaTable.getConfiguration()
+      .get(LOCATOR_META_REPLICAS_MODE, CatalogReplicaMode.NONE.toString()));
+
+    if (metaReplicaMode == CatalogReplicaMode.LOAD_BALANCE) {
+      addListener(metaTable.getDescriptor(), (desc, error) -> {
+        if (error != null) {
+          LOG.warn("Failed to get meta table descriptotr, error={}", error);
```

Comment:
Here we do not want to log the stacktrace so we use error={}?

Reply:
We need the stacktrace, update in the new patch.